src/amqp_management.c

```diff
@@ -281,14 +279,53 @@ static void on_message_sender_state_changed(void* context, MESSAGE_SENDER_STATE
         break;
 
     case MESSAGE_SENDER_STATE_OPEN:
-        amqp_management_instance->sender_connected = 1;
-        (void)send_operation_messages(amqp_management_instance);
+        switch (amqp_management_instance->amqp_management_state)
+        {
+        default:
+        case AMQP_MANAGEMENT_STATE_ERROR:
+            /* do nothing */
+            break;
+
+        case AMQP_MANAGEMENT_STATE_OPENING:
+            amqp_management_instance->sender_connected = -1;
```

Comment:
```amqp_management_instance->sender_connected = -1``` doesn't make sense if you are testing it right below.

What's the intent here?